modnet/fit_genetic.py

```diff
@@ -157,14 +223,12 @@ def function_fitness(
             baseline=None,
             restore_best_weights=True,
         )
-        callbacks = [rlr, es]
+        callbacks = [es]
         for w in self.pop:
-            modnet_model = MODNetModel([[['BV_Ea']]], {'BV_Ea':1}, n_feat=w[0], num_neurons=[[int(w[1])],[int(w[1]*w[2])],[int(w[1]*w[2]*w[3])],[int(w[1]*w[2]*w[3]*w[4])]], act=w[5])
+            modnet_model = MODNetModel([[[X_train.df_targets.columns[0]]]], {X_train.df_targets.columns[0]:1}, n_feat=w[0], num_neurons=[[int(w[1])],[int(w[1]*w[2])],[int(w[1]*w[2]*w[3])],[int(w[1]*w[2]*w[3]*w[4])]], act=w[5])
```

Comment:
we should think to adapt to multi-property models in the future (or maybe make a different genetic class for that as it the architecture space is quite different). And this could be powerful, because I think joint-learning with the right hyperparameters can have a significant gain w.r.t single learning.